src/server/routes/pc.js

```diff
@@ -10,12 +10,17 @@ router.get('/datasources', function (req, res) {
 });
 
 router.get('/querySearch', function (req, res) {
-  pc.querySearch(req.query).then(r => res.json(r));
+  pc.search(req.query).then(r => res.json(r));
 });
 
 //this is mainly to download and save raw data (with correct content-time) from PC ws directly to client
-router.get('/:path', function (req, res) {
-  res.redirect(config.PC_URL + req.params.path + '?' + qs.stringify(req.query));
+router.get('/pc2/:path', function (req, res) {
+  res.redirect(config.PC_URL + 'pc2/' + req.params.path + '?' + qs.stringify(req.query));
```

Comment:
This might be looking for the 'pc2' stripping it, then adding back?. Also could this just point to pc.query method?

Reply:
initially, router.get('/:path', function (req, res) - was to match/route ALL other possible direct requests to PC2 - mainly (or only) for the GSEA/SIF/SBGN/.. download links in the view's right-side panel.
I just had to made this change after changes in pc (server/pathway-commons) module, which redefined what 'cmd' (command) means - now it means full context path, such as pc2/search or sifgraph/v1/pathsbetween